fdbsyncd/fdbsync.cpp

```diff
@@ -43,6 +43,66 @@ FdbSync::~FdbSync()
     }
 }
 
+
+// Check if interface entries are restored in kernel
+bool FdbSync::isIntfRestoreDone()
+{
+    vector<string> required_modules = {
+            "vxlanmgrd",
+            "intfmgrd",
+            "vlanmgrd",
+            "vrfmgrd"
+        };
+
+    for(string& module : required_modules)
+    {
+        WarmStart::WarmStartState state;
+        
+        WarmStart::getWarmStartState(module, state);
+        if(state == WarmStart::REPLAYED || state == WarmStart::RECONCILED)
+        {
+            SWSS_LOG_INFO("Module %s Replayed or Reconciled %d",module.c_str(), (int) state);            
+        }
+        else
+        {
+            SWSS_LOG_INFO("Module %s NOT Replayed or Reconciled %d",module.c_str(), (int) state);            
+            //return false;
+            //Return true till all the dependant code is checked in
+            return true;
+        }
+    }
+    
+    return true;
+}
+
+// Check if vxlan entries are re-conciled to kernel 
+bool FdbSync::isReadyToReconcile()
+{
+    vector<string> required_modules = {
+            "orchagent",
+        };
+
+    for(string& module : required_modules)
+    {
+        WarmStart::WarmStartState state;
+        
+        WarmStart::getWarmStartState(module, state);
+        if(state == WarmStart::RECONCILED)
+        {
+            SWSS_LOG_INFO("Module %s Reconciled %d",module.c_str(), (int) state);            
+        }
+        else
+        {
+            SWSS_LOG_INFO("Module %s NOT Reconciled %d",module.c_str(), (int) state);            
+            //return false;
+            //Return True untill the dependant orchagent code is commited

```

Comment:
The same

Reply:
This code and comment will be deleted when all the dependent warm-reboot code is available in the code base. will add the sapce though